scale-tests/deploy-dispatchers.py

```diff
@@ -170,12 +166,15 @@ def deploy_dispatchers(
         }
     }
 
+    options_file = arguments['--options-json']
+    if options_file:
+        options = json.load(open(options_file, 'r'))
```

Comment:
Makes sense using `with open(...)` to close the file?

Reply:
For a single open, in a short-running program, *and* read-only access I think putting it on one line is fine.